docs/src/documentation/README.md

```diff
@@ -0,0 +1,22 @@
+# Einführung
+
+Die ISI Anwendung wird in der Microservice Architektur umgesetzt und in Github als Multi-Repos verwaltet.
+
+Derzeit werden die folgenden Services in Github unter der MIT Lizenz Open Source zur Verfügung gestellt:
+
+## ISI Frontend
+Das [ISI Frontend](https://github.com/it-at-m/isi-frontend) liefert die Weboberfläche der ISI Anwendung aus, enthält
```

Comment:
Im Repo für das Frontend und Backend liegt jeweils die selbe Beschreibung.

Alternativ könnten wir die docs in ein eigenes isi-Repo packen. z.B. `isi-docs`

In den Readmes des Backends und Frontends kann dann auf das Repo isi-docs verwiesen werden.

Reply:
passt